docs/aiactionunsafesandbox.md

```
@@ -0,0 +1,132 @@
+---
+title: "AI Action Unsafe Sandbox Rule"
+weight: 1
+---
+
+### AI Action Unsafe Sandbox Rule Overview
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)